docker/ubuntu-gtsam/Dockerfile

```diff
@@ -0,0 +1,35 @@
+# Get the base Ubuntu image from Docker Hub
+FROM dellaert/ubuntu-boost-tbb-eigen3:bionic
+
+# Install git
+RUN apt-get update && \ 
+    apt-get install -y git
+
+# Install compiler
+RUN apt-get install -y build-essential
+
+# Clone GTSAM
+WORKDIR /usr/src/
+RUN git clone https://bitbucket.org/gtborg/gtsam.git
```

Comment:
Needs to be updated to use `github` and perhaps pull the `develop` branch which has a lot of new niceness?